plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -82,177 +82,391 @@ function executeCLI($scriptName, &$args) {
+				$submissionFile->setGenreId($genre->getId());
+				$submissionFile->setFileStage(SUBMISSION_FILE_PROOF);
+				$submissionFile->setAssocType(ASSOC_TYPE_REPRESENTATION);
+				$submissionFile->setData('assocId', $publicationFormatId);
+				$submissionFile->setData('createdAt', Core::getCurrentDate());
+				$submissionFile->setDateModified(Core::getCurrentDate());
+				$submissionFile->setData('mimetype', $mimeType);
+				$submissionFile->setData('fileId', $fileId);
+				$submissionFile->setData('name', pathinfo($filePath, PATHINFO_FILENAME));
+
+				// Assume open access, no price.
+				$submissionFile->setDirectSalesPrice(0);
+				$submissionFile->setSalesType('openAccess');
+
+				$submissionFileDao->insertObject($submissionFile);
+
+				// Keywords
+				$keywordsList = [$locale => explode(';', $keywords)];
+				if (count($keywordsList[$locale]) > 0) {
+					/** @var $submissionKeywordDao SubmissionKeywordDAO */
+					$submissionKeywordDao = DAORegistry::getDAO('SubmissionKeywordDAO');
+					$submissionKeywordDao->insertKeywords($keywordsList, $publicationId);
 				}
+
+				//Subjects
+				$subjectsList = [$locale => explode(';', $subjects)];
+				if (count($subjectsList[$locale]) > 0) {
+					/** @var $submissionKeywordDao SubmissionKeywordDAO */
+					$submissionSubjectDao = DAORegistry::getDAO('SubmissionSubjectDAO');
+					$submissionSubjectDao->insertSubjects($subjectsList, $publicationId);
+				}
+
+				$coverImagelocale = [];
+				$coverImage = [];
+
+				if ($bookCoverImageName) {
+					$sanitizedCoverImageName = str_replace([' ', '_', ':'], '-', mb_strtolower($bookCoverImageName));
+					$sanitizedCoverImageName = PKPstring::regexp_replace("/[^a-z0-9\.\-]+/", '', $sanitizedCoverImageName);
+					$sanitizedCoverImageName = basename($sanitizedCoverImageName);
+
+					$coverImage['uploadName'] = uniqid() . '-' . $sanitizedCoverImageName;
+					$coverImage['altText'] = $bookCoverImageAltText ?? '';
+
+					$destFilePath = $publicFileManager->getContextFilesPath($press->getId()) . '/' . $coverImage['uploadName'];
+					file_put_contents($destFilePath, file_get_contents($srcFilePath));
+
+					$publicationService->makeThumbnail(
+						$destFilePath,
+						$publicationService->getThumbnailFileName($coverImage['uploadName']),
+						$press->getData('coverThumbnailsMaxWidth'),
+						$press->getData('coverThumbnailsMaxHeight')
+					);
+
+					$coverImagelocale[$locale] = $coverImage;
+
+					$publication->setData('coverImage', $coverImagelocale);
+					$publicationDao->updateObject($publication);
+				}
+
+				// Categories
+
+				foreach ($dbCategories as $category) {
+					$categoryDao->insertPublicationAssignment($category->getId(), $publicationId);
+				}
+
+				echo __('plugins.importexport.csv.import.submission', ['title' => $title]) . "\n";
+			}
+
+			if (count($invalidRows) > 0) {
+				$csvForInvalidRowsName = "{$basePath}/invalid_rows.csv";
+				$csvDataWithHeaders = [...$invalidRows];
```

Comment:
The `stable-3_3_0` should work under PHP 7.3, this isn't supported.

Reply:
I just removed this section as I'm working with an updated flow now that I removed the $invalidRows as you suggested.